### PR TITLE
MAB-related updates

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP10CNSM/CP10CNSM_D00001_ingest.csv [hunk 1]
```diff
@@ -26,7 +26,7 @@ mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OM
 mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl27/flort/*.flort.log,CP10CNSM-RID27-02-FLORTD000,telemetered,Available,
 mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP10CNSM-RID27-03-CTDBPC000,telemetered,Available,
 mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl27/dosta1/*.dosta1.log,CP10CNSM-RID27-04-DOSTAD000,telemetered,Available,
-mi-instrument.mi.dataset.driver.plims_a.plims_a_hdr_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl27/plims/*.plims.log,CP10CNSM-RID27-07-PLIMSA000,telemetered,Available,
+mi.dataset.driver.plims_a.plims_a_hdr_driver,/omc_data/whoi/OMC/CP10CNSM/D00001/cg_data/dcl27/plims/*.hdr,CP10CNSM-RID27-07-PLIMSA000,telemetered,Available,
```
Comment (cooleyky) [inline CP10CNSM/CP10CNSM_D00001_ingest.csv:29]: looks good

File: CP11SOSM/CP11SOSM_D00001_ingest.csv [hunk 2]
```diff
@@ -8,7 +8,7 @@ mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OM
 mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl36/pco2w/*.pco2w.log,CP11SOSM-MFD36-05-PCO2WB000,telemetered,Available,
 mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl36/phsen2/*.phsen2.log,CP11SOSM-MFD36-06-PHSEND000,telemetered,Available,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/syslog/*.syslog.log,CP11SOSM-MFD37-00-DCLENG000,telemetered,Available,
-#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP11SOSM-MFD37-01-OPTAAD000,telemetered,Not Expected,Failed on deployment 
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP11SOSM/D00001/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP11SOSM-MFD37-01-OPTAAD000,telemetered,Available,
```
Comment (cooleyky) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:11]: looks good, data shown in raw data repo

**Review by cooleyky — APPROVED** [review]
looks good

File: CP10CNSM/CP10CNSM_D00001_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [inline CP10CNSM/CP10CNSM_D00001_ingest.csv:29]: looks good

File: CP11SOSM/CP11SOSM_D00001_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [inline CP11SOSM/CP11SOSM_D00001_ingest.csv:11]: looks good...it's ALIVE!

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**